packages/edit-site/src/components/start-template-options/index.js

```diff
@@ -50,6 +50,37 @@ function useStartPatterns( fallbackContent ) {
 		};
 	}, [] );
 
+	const currentThemeStylesheet = useSelect(
+		( select ) => select( coreStore ).getCurrentTheme().stylesheet
+	);
+
+	// Duplicated from packages/block-library/src/pattern/edit.js.
```

Comment:
should we make a similar comment in that file so any future changes get ported to this function too?